brjs-core/src/main/java/org/bladerunnerjs/model/NodeImporter.java

```diff
@@ -205,22 +205,30 @@ private static void findAndReplaceInTextFiles(BRJS brjs, Collection<File> files,
 	{
 		for (File f : files) {
 			if (f.length() != 0) {
-				if (checkFileMimeType(f).startsWith("text")) {
-					findAndReplaceInTextFile(brjs, f, sourceRequirePrefix, targetRequirePrefix);
+				try {
+					if (checkFileMimeType(f).startsWith("text")) {
+						findAndReplaceInTextFile(brjs, f, sourceRequirePrefix, targetRequirePrefix);
+					}
+				}
+				catch (MagicMatchNotFoundException e) {
+					continue;
 				}
 			}
 		}
 	}
 
-	private static String checkFileMimeType(File file) throws IOException {
+	private static String checkFileMimeType(File file) throws IOException, MagicMatchNotFoundException {
 		byte[] data = Files.readAllBytes(Paths.get(file.getAbsolutePath()));
```

Comment:
@andyberry88, I think your idea is sound, but I don't think it's particularly high priority since [jmimemagic](https://github.com/arimus/jmimemagic) is actually quite efficient in that it only scans the beginning of the file, and given that importing apps is only performed occasionally and is not a time critical operation.